src/SprykerEco/Zed/Adyen/AdyenConfig.php

```diff
@@ -557,7 +557,7 @@ public function getRequestChannel(): string
     /**
      * @api
      *
-     * @return string[]
+     * @return array<string>
```

Comment:
If we define the specific keys, we should describe key type as well `array<string, string>`

Reply:
Fixed.